libs/dh/shared/util-apollo/src/lib/query.ts

```diff
@@ -154,6 +155,7 @@ export function query<TResult, TVariables extends OperationVariables>(
   // Signals holding the result values
   const data = signal<TResult | undefined>(initial.data);
   const error = signal<ApolloError | undefined>(initial.error);
+  const hasError = signal(false);
```

Comment:
question: since this property is derived from the value of `error`, maybe we can make it a `computed` instead?

suggestion: adjust the tests to include this property has well.

Reply:
Good ideas, now implemented